src/3d/qgscameracontroller.h

```diff
@@ -242,6 +243,39 @@ class _3D_EXPORT QgsCameraController : public QObject
      */
     bool hasInputHandlersEnabled() const { return mInputHandlersEnabled; }
 
+    /**
+     * Orbits camera around the globe by the specified amount given as the difference
+     * in latitude/longitude angles (in degrees)
+     * \since QGIS 3.44
+     */
+    void globeMoveCenterPoint( double latDiff, double lonDiff );
+
+    /**
+     * Moves camera closer or further away from the globe. Use factor greater than one
+     * to zoom in, or less than one to zoom out.
+     * \since QGIS 3.44
+     */
+    void globeZoom( float factor );
+
+    /**
+     * Updates pitch angle by the specified amount given as the angular difference in degrees
+     * \since QGIS 3.44
+     */
+    void globeUpdatePitchAngle( float angleDiff );
+
+    /**
+     * Updates heading angle by the specified amount given as the angular difference in degrees
+     * \since QGIS 3.44
+     */
+    void globeUpdateHeadingAngle( float angleDiff );
+
+    /**
+     * Resets view of the globe to look at a particular location given as latitude
+     * and longitude (in degrees) and distance from that point on ellipsoid
+     * \since QGIS 3.44
+     */
+    void resetGlobe( float distance, double lat = 0, double lon = 0 );
```

Comment:
What would you think about changing the parameters' order to use `lon` first, `lat` second?
Matching the x-y order feels more natural to me.

Reply:
i'm easy... I was thinking I should probably switch from lat/lon min/max to QgsRectangle and from lat/lon to QgsPointXY - to avoid any confusion